apps/webapp/app/presenters/v3/SpanPresenter.server.ts

```
@@ -376,12 +423,41 @@ export class SpanPresenter extends BasePresenter {
+          select: {
+            friendlyId: true,
+            type: true,
+            status: true,
+            idempotencyKey: true,
+            userProvidedIdempotencyKey: true,
+            idempotencyKeyExpiresAt: true,
+            output: true,
+            outputType: true,
+            outputIsError: true,
+          },
+        });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)